tests/Feature/Controllers/ArtistControllerTest.php

```diff
@@ -19,6 +31,13 @@ public function setUp()
         $this->country = resolve(CountryRepositoryInterface::class);
         $this->artist = resolve(ArtistRepositoryInterface::class);
         $this->profile = resolve(ProfileRepositoryInterface::class);
```

Comment:
Shouldn't all 3 of these properties be initialized above just like the below ones ?

Reply:
Yes; but a bunch of tests need the same type of clean-up, so I'm going to handle them in a single commit.